scripts/whitepaper.css

```
@@ -0,0 +1,41 @@
+@page { size: A4; margin: 22mm 20mm 20mm; }
```

Stylelint (17.14.0)

[error] Expected "A4" to be "a4" (value-keyword-case)

(value-keyword-case)